web/src/compose_setup.ts

```diff
@@ -376,10 +389,10 @@ export function initialize() {
             return;
         }
 
-        compose_call_ui.generate_and_insert_audio_or_video_call_link($(e.target), false);
+        compose_call_ui.generate_and_insert_audio_or_video_call_link($(this), false);
```

Comment:
`generate_and_insert_audio_or_video_call_link` only uses `parents` on the `$target_element`, so this should be equivalent